vllm/distributed/parallel_state.py

```diff
@@ -376,3 +387,9 @@ def destroy_model_parallel():
     _PP_DEVICE_GROUP = None
     global _PP_GLOBAL_RANKS
     _PP_GLOBAL_RANKS = None
+    global _PP_CPU_GROUP
+    if _PP_CPU_GROUP:
```

Comment:
what is this group for?

Reply:
No longer relevant in this version - it is basically for metadata sending/recving though